### ISSUE TITLE
Add `?ifnot` to preprocessor

### ISSUE THREAD

**Issue by ChHaeni** [oposs/agrammon#283]
Currently, there are a few code blocks that need to be parsed for all variants except for one variant... An "ifnot ..." statement would reduce code and improve readability compared to an 'if ... else ...' statement.

**Comment by ChHaeni** [comment]
It seems that I can use `?if` with several model variants, e.g. `?if variant1,variant2,variant3`. If so, I'm fine with omitting this request. However, one big advantage of 'ifnot' over 'if v1,v2' would be that one doesn't need to search for existing if statements to extend them when adding a new variant.

**Comment by ChHaeni** [comment]
> It seems that I can use `?if` with several model variants, e.g. `?if variant1,variant2,variant3`. If so, I'm fine with omitting this request. However, one big advantage of 'ifnot' over 'if v1,v2' would be that one doesn't need to search for existing if statements to extend them when adding a new variant.

Ok, it actually doesn't seem to be possible to have more than one variant after an `?if` statement.

**Comment by ChHaeni** [comment]
@jnthn is it as simple as adding
```
                when /^ '?ifnot' \h+ <option=.ident> \h* $/ {
                    my $enabled = not so %options{~$<option>};
                    my $matched = $enabled;
                    my $start-line = $number + 1;
                    @open.push(OpenDirective.new(:$start-line, :$enabled, :$matched, :accept-elsif));
                }
```
to Preprocessor.pm6?

**Comment by jnthn** [comment]
Resolved by merge of #291.
